### PR TITLE
Add a simple How-To motivated by current issues

### PR DESCRIPTION
**Pull request by kunaltyagi**
Motivated by #1462 and #2557

Comments welcome :)

### CODE REVIEW

File: docs/HowDoI_interface.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# How do I ?
```
Comment (apmorton) [inline docs/HowDoI_interface.md:1]: maybe this should be index.md?

I believe github renders index.md by default when you browse to a folder

**Review by RubenRBS — CHANGES_REQUESTED** [review]
Thanks you for taking the time to do this ❤️

I can't leave a review comment on the images, so I have to use this message instead:
Could the text on the images be changed so the font is a bit bigger? Ideally, it could also have a dark background and white text color (I'm thinking about something like [bootstrap tooltips](https://getbootstrap.com/docs/4.0/components/tooltips/#examples)) so that the constrast is greater. Right now, I find it a bit hard to read :)

File: docs/HowDoI_interface.md (hunk 1 above)
Comment (kunaltyagi) [reply docs/HowDoI_interface.md:1]: I don't see any such feature with [the official cheatsheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf)

I did find [this cli utility](https://github.com/ekalinin/github-markdown-toc#auto-insert-and-update-toc), but not anything related to GitHub. Could you please link the feature you referenced?

**Comment by kunaltyagi** [comment]
> Could the text on the images be changed so the font is a bit bigger?

I just used flameshot for this and this feature [hasn't landed yet](https://github.com/flameshot-org/flameshot/issues/1249). Let me see if I can instead use an editing software to make the necessary changes.

**Comment by Samuel-Martin23** [comment]
@kunaltyagi Thanks for checking out the issue. I added the shadow effect but I just need to find a way of adding a border around the text.

**Comment by partouf** [comment]
Merging this to get something going, you can send another PR to change things if you want.

File: docs/HowDoI_interface.md (hunk 1 above)
Comment (partouf) [reply docs/HowDoI_interface.md:1]: apparently you need to call it readme.md, index doesn't work

**Pull request merged**